### PR TITLE
Per mob "Reaction Times"

### PR DESCRIPTION
**Pull request by RemieRichards**
<B> Code: </B>
- Adds `next_move_adjustment`, a decisecond + or - value that modifies how much click/action/interaction delays affect a mob
- Adds `next_move_modifier`, a multiplier value that modifies how much click/action/interaction delays affect a mob

<B>Examples (NOT IMPLEMENTED):</B>
- A mob that does very little damage, but has 0 click delay, the speed is then based on the user/AI's delay
- Vampires being twice as fast as people in reaction times, like they sometimes are in other media.

### CODE REVIEW

**Comment by Anonus** [comment]
Oooh, that'd be really cool for vampires.
Probably a bad idea if it affected guns but it would be really cool, watching a vampire beat you to death with fists alone.

**Comment by WJohn** [comment]
Oh man that's really cool. It'd be cool to see this be played with with certain speed-boosting chems. Maybe make them also reduce click delay while you have them in you.

**Comment by KorPhaeron** [comment]
@Anonus 

I think the gun has its own click delay

**Comment by RemieRichards** [comment]
@KorPhaeron same proc, so it would affect it yes.

**Comment by KorPhaeron** [comment]
Huh, might be pretty strong then. Will be interesting to see. Though I'm guessing it wont be as powerful as it sounds in melee combat due to everyone running around like headless chickens and taking passing swipes instead of standing face to face.

Will definitely look into adding this to different things once it's in though.

**Comment by PKPenguin321** [comment]
can badminry modify click delay to 0

**Comment by RemieRichards** [comment]
@PKPenguin321 badmins have access to all vars on /atom upwards, and sometimes /datum vars.
Mobs are /atom/movable/mob.

tl;dr yes.

**Comment by PKPenguin321** [comment]
okay, well then after this is in we need an adminbutton that makes everybody go super fast (pre speed nerf level fast) and have 0 click delay

**Comment by RemieRichards** [comment]
@PKPenguin321 no need, next to every var is the M option, which stands for "Mass Modify" changing it on all things in the world that also have that var.
